### ISSUE TITLE
[Proposal] Выбор роли лидера в меню антагов

### ISSUE THREAD

**Issue by Smartcookie912** [TauCetiStation/TauCetiClassic#11649]
Добавить возможность в меню выбора антагонистов занимать/не занимать главную роль. Например быть лидером/рядовым нюки, лидером культа или просто прислужником и т.д.

**Comment by AndroBetel** [comment]
это, во-первых, затруднит выбор игроков на роли выбора, из-за чего тупо режим может не стартоваться, а во-вторых - никакого смысла в этом нет, потому что лидер нюки может просто поделиться доступом с желающим игроком, а единственная обязанность лидера культа - выбрать раундстартом аспекты

**Comment by Smartcookie912** [comment]
> это, во-первых, затруднит выбор игроков на роли лидера, из-за чего тупо режим может не стартоваться, а во-вторых - никакого смысла в этом нет, потому что лидер нюки может просто поделиться доступом с желающим игроком, а единственная обязанность лидера культа - выбрать раундстартом аспекты

Да, но бывают новички которые каждый раз буквально пишут _Ой ребята я никогда не командовал сделайте кто-то другой_, и в случае культа приходится через лоос говорить куда жать

**Comment by pro100kotah** [comment]
Ну, Бетель все сказал. У нас из-за такой новой функции просто раунд стартоваться не будет, так как не был забит слот. Можно, конечно, сделать приоритетность, а когда нет желающих брать рандомно, как есть сейчас. Но даже не знаю, кто будет ради такой мороки заниматься впилом ветки кода.

**Comment by DarkFurrSss** [comment]
> это, во-первых, затруднит выбор игроков на роли лидера, из-за чего тупо режим может не стартоваться, а во-вторых - никакого смысла в этом нет, потому что лидер нюки может просто поделиться доступом с желающим игроком, а единственная обязанность лидера культа - выбрать раундстартом аспекты

А если такими префами лишь значительно повысить шанс выпадения? Например, если не вышло в нюке найти челика с префом на командира, то выпадет рандом. И т.п.

**Comment by Deahaka** [comment]
Если никто не хочет быть лидером, то никому и не надо давать лидера. И режим пусть не запустится.

**Comment by Oz1Es** [comment]
Почему деахака козлик?

**Comment by Smartcookie912** [comment]
🤷🏼‍♀️

**Comment by Deahaka** [comment]
Актуальности не потеряло
